From 4e41bb64f6e883526ed4ad8494a286353669cd4b Mon Sep 17 00:00:00 2001
From: Deepa G K <g.k.deepa@intel.com>
Date: Mon, 29 Feb 2016 15:49:07 +0530
Subject: [PATCH 2/3] Asoc: rt5645: Klockwork fix for Unvalidated integer value

Klockwork reported an Issue where 'm_t' might be used uninitialized
in the function rt5645_pll_calc()

Change-Id: I27c9f657fe4b7eee14579b94b257bef148cd1ced
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-21889
Signed-off-by: Deepa G K <g.k.deepa@intel.com>
Signed-off-by: Kishore Kadiyala <kishore.kadiyala@intel.com>
Reviewed-on: https://android.intel.com:443/475710
---
 sound/soc/codecs/rt5645.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/rt5645.c b/sound/soc/codecs/rt5645.c
index 45942f3..ef44475 100644
--- a/sound/soc/codecs/rt5645.c
+++ b/sound/soc/codecs/rt5645.c
@@ -2701,7 +2701,7 @@ static int rt5645_pll_calc(const unsigned int freq_in,
 	const unsigned int freq_out, struct rt5645_pll_code *pll_code)
 {
 	int max_n = RT5645_PLL_N_MAX, max_m = RT5645_PLL_M_MAX;
-	int k, red, n_t, m_t, pll_out, in_t, out_t;
+	int k, red, n_t, m_t = 0, pll_out, in_t, out_t;
 	int red_t = abs(freq_out - freq_in);
 	bool bypass = false;
 	int n = 0, m = 0;
-- 
1.9.1

